package.json

```diff
@@ -35,7 +35,8 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.2.1",
     "@rollup/plugin-replace": "^5.0.2",
-    "@rollup/plugin-typescript": "11.1.3",
+    "@rollup/plugin-typescript": "12.1.1",
+    "@types/bignumber.js": "^5.0.4",
```

Comment:
You should probably delete this line. bignumber.js types are distributed with the main package now, so these are 4 major versions out of date.